lib/propshaft/railtie.rb

```diff
@@ -12,6 +12,7 @@ class Railtie < ::Rails::Railtie
       [ "text/javascript", Propshaft::Compilers::SourceMappingUrls ]
     ]
     config.assets.sweep_cache = Rails.env.development?
+    config.assets.server = !Rails.env.production?
```

Comment:
Think we need the opposite here. Turning the server on in dev + test, so it'll remain off in other envs like staging and beta.

Reply:
https://github.com/rails/propshaft/pull/62